src/yacfg/profiles/artemis/2.18.0/showcase.yaml.jinja2

```diff
@@ -0,0 +1,715 @@
+    - tcpReceiveBufferSize: 1048576
+    - protocols: AMQP
+    - useEpoll: true
+    - amqpCredits: 1000
+    - amqpLowCredits: 300
+    - saslMechanisms: GSSAPI
+    - saslLoginConfigScope: mykerberos
+
+
+  security_settings:
+    - match: '#'
+      permissions:
+        createNonDurableQueue:
+        - amq
+        - test
+        - asdf
+        deleteNonDurableQueue:
+        - amq
+        createDurableQueue:
+        - amq
+        deleteDurableQueue:
+        - amq
+        createAddress:
+        - amq
+        deleteAddress:
+        - amq
+        consume:
+        - amq
+        browse:
+        - amq
+        send:
+        - amq
+        manage:
+        - amq
+    - class_name: org.apache.activemq.artemis.core.server.impl.LegacyLDAPSecuritySettingPlugin
+      properties:
+      - initialContextFactory: com.sun.jndi.ldap.LdapCtxFactory
+      - connectionURL: ldap://localhost:10389
+      - connectionUsername: uid=admin,ou=system
+      - connectionPassword: secret
+      - connectionProtocol: s
+      - authentication: simple
+
+
+  address_settings:
+  - match: activemq.management#
+    dead_letter_address: DLQ
+    expiry_address: ExpiryQueue
+    redelivery_delay: 0
+    redistribution_delay: 0
+    max_size_bytes: -1
+    message_counter_history_day_limit: 10
+    address_full_policy: PAGE
+    auto_create_queues: true
+    auto_create_addresses: true
+    auto_create_jms_queues: true
+    auto_create_jms_topics: true
+  - match: '#'
+    dead_letter_address: DLQ
+    expiry_address: ExpiryQueue
+    redelivery_delay: 0
+    max_size_bytes: -1
+    message_counter_history_day_limit: 10
+    address_full_policy: PAGE
+    auto_create_queues: true
+    auto_create_addresses: true
+    auto_create_jms_queues: true
+    auto_create_jms_topics: true
+
+  addresses:
+  - name: DLQ
+    anycast:
+    - name: test
+      type: queue
+      max_consumers: 10
+      exclusive: true
+    - name: jinaqueue
+      type: queue_block
+      max_consumers: 10
+      purge_on_no_consumers: false
+      durable: true
+      filter: asdf=qwer
+      exclusive: false
+    multicast:
+    - name: jinaqueue2
+      type: queue
+      durable: false
+
+  - name: ExpiryQueue
+    anycast:
+    - name: ExpiryQueue
+      type: queue
+    multicast: true
+
+  - name: pubsub.foo
+    multicast: true
+
+  broker_plugins:
+  - class: org.apache.activemq.artemis.core.server.plugin.impl.LoggingActiveMQServerPlugin
+    properties:
+    - LOG_ALL_EVENTS: true
+    - LOG_CONNECTION_EVENTS: true
+    - LOG_SESSION_EVENTS: true
+    - LOG_CONSUMER_EVENTS: true
+    - LOG_DELIVERING_EVENTS: true
+    - LOG_SENDING_EVENTS: true
+    - LOG_INTERNAL_EVENTS: true
+  - class: org.apache.activemq.artemis.core.server.plugin.impl.LoggingActiveMQServerPlugin2
+
+
+# cluster related configurations
+  connectors:
+    - name: artemis
+      protocol: tcp
+      address: 10.0.0.1
+      port: 61616
+    - name: test2
+      protocol: tcp
+      address: 172.18.0.1
```

Comment:
This used to be `172.17.0.1` in artemis 2.17.0 profile. Is that change intentional?

Reply:
Thank @jiridanek. You have eagle eyes! :) The change is unintentional. I use search replace after copy the profile:
/2.17.0/2.18.0/g
so 17**2.17.0**.1 becomes 17**2.18.0**.1
I'll correct it. Thanks again!